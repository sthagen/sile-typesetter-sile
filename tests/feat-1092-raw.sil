\begin[papersize=a6]{document}
\use[module=packages.retrograde,target=v0.15.0]
\nofolios
\neverindent
\use[module=packages.svg]
% Test Case 001: Direct use of a raw SVG.
\raw[type=svg, height=1em]{<?xml version="1.0"?>
<svg viewBox="0 0 319.99756 192.49629"><path d="M 160.36936,64.6995 287.47181,192.49629 319.99756,160.32293 160.62175,0 0,159.07464 32.273754,191.50079 Z"/></svg>}

% Test Case 002: Raw SVG from a macro, with re-use.
\define[command=test:svg]{\raw[type=svg, height=1em]{<?xml version="1.0"?>
<svg viewBox="0 0 319.99756 192.49629"><path d="M 160.36936,64.6995 287.47181,192.49629 319.99756,160.32293 160.62175,0 0,159.07464 32.273754,191.50079 Z"/></svg>}}

\test:svg \test:svg

% Test Case 003: Direct use of an raw plain text.
\raw[type=text]{
This is a plain text inline.
  A bit like a true verbatim.
\lets[do]{weird things}% Yeah!
}
\end{document}
