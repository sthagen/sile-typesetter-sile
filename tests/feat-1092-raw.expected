Set paper size 	297.6377985	419.5275636
Begin page
Draw SVG	30.9764	16.6236	10.0000	0.0519	160.369354 64.699501 m 202.736832 107.298431 245.104324 149.897369 287.471802 192.496292 c 298.313721 181.771835 309.155640 171.047394 319.997559 160.322937 c 266.872284 106.881958 213.747025 53.440979 160.621750 0.000000 c 107.081161 53.024883 53.540585 106.049759 0.000000 159.074646 c 10.757918 169.883362 21.515835 180.692078 32.273754 191.500793 c 74.972290 149.233704 117.670822 106.966599 160.369354 64.699501 c 0.000000 0.000000 0.000000 rg h f 
Draw SVG	42.9764	16.6236	10.0000	0.0519	160.369354 64.699501 m 202.736832 107.298431 245.104324 149.897369 287.471802 192.496292 c 298.313721 181.771835 309.155640 171.047394 319.997559 160.322937 c 266.872284 106.881958 213.747025 53.440979 160.621750 0.000000 c 107.081161 53.024883 53.540585 106.049759 0.000000 159.074646 c 10.757918 169.883362 21.515835 180.692078 32.273754 191.500793 c 74.972290 149.233704 117.670822 106.966599 160.369354 64.699501 c 0.000000 0.000000 0.000000 rg h f 
Draw SVG	42.9764	16.6236	10.0000	0.0519	160.369354 64.699501 m 202.736832 107.298431 245.104324 149.897369 287.471802 192.496292 c 298.313721 181.771835 309.155640 171.047394 319.997559 160.322937 c 266.872284 106.881958 213.747025 53.440979 160.621750 0.000000 c 107.081161 53.024883 53.540585 106.049759 0.000000 159.074646 c 10.757918 169.883362 21.515835 180.692078 32.273754 191.500793 c 74.972290 149.233704 117.670822 106.966599 160.369354 64.699501 c 0.000000 0.000000 0.000000 rg h f 
Mx 	14.8819
My 	54.9764
Set font 	Gentium Plus;10;400;;normal;;;LTR;
T	55 75 76 86 w=17.5977	(This)
Mx 	35.1400
T	76 86 w=6.5723	(is)
Mx 	44.3727
T	68 w=4.5898	(a)
Mx 	51.6230
T	83 79 68 76 81 w=20.8252	(plain)
Mx 	75.1087
T	87 72 91 87 w=16.5332	(text)
Mx 	94.3023
T	76 81 79 76 81 72 w=23.7842	(inline)
Mx 	118.0865
T	17 w=2.2900	(.)
Mx 	20.2013
My 	66.9764
T	36 w=5.9521	(A)
Mx 	28.8132
T	69 76 87 w=11.2305	(bit)
Mx 	42.7033
T	79 76 78 72 w=14.9170	(like)
Mx 	60.2801
T	68 w=4.5898	(a)
Mx 	67.5296
T	87 85 88 72 w=17.3193	(true)
Mx 	87.5087
T	89 72 85 69 68 87 76 80 w=37.3633	(verbatim)
Mx 	124.8719
T	17 w=2.2900	(.)
Mx 	14.8819
My 	78.9764
T	63 w=4.6924	(\)
Mx 	19.5743
T	79 72 87 86 w=14.6338	(lets)
Mx 	34.2081
T	62 w=3.0762	([)
Mx 	37.2842
T	71 82 w=10.2295	(do)
Mx 	47.5137
T	64 w=3.0762	(])
Mx 	50.5899
T	94 w=3.3594	({)
Mx 	53.9493
T	90 72 76 85 71 w=23.5498	(weird)
Mx 	80.1571
T	87 75 76 81 74 86 w=25.9277	(things)
Mx 	106.0848
T	96 w=3.3594	(})
Mx 	109.4442
T	8 w=6.8994	(%)
Mx 	119.0017
T	60 72 68 75 w=20.6689	(Yeah)
Mx 	139.6706
T	4 w=2.7197	(!)
End page
Finish
